WalletWasabi.Packager/Program.cs

```diff
@@ -396,6 +399,103 @@ private static void Main(string[] args)
 							tarProcess.WaitForExit();
 						}
 
+						Console.WriteLine("Create Linux .deb");
+
+						var debFolderRelativePath = "deb";
+						var debFolderPath = Path.Combine(binDistDirectory, debFolderRelativePath);
+						var linuxUsrBinFolder = "/usr/bin/";
```

Comment:
According to [this](http://www.pathname.com/fhs/2.2/) the installation folder should be `/usr/local/bin/`. I check my system just to be sure and yes, most of the software is installed under `/usr/local/bin/`.